src/main/java/sx/blah/discord/api/internal/DiscordVoiceWS.java

```diff
@@ -257,11 +257,13 @@ public void run() {
 	 * Sets up the thread for receiving audio packets from the Discord UDP Socket.
 	 */
 	private void setupReceiveThread() {
-		Runnable receiveThread = () -> {
+		(new Thread((Runnable) () -> {
```

Comment:
And using a raw thread as opposed to an executor service will make the jvm never stop even if the user logs out.